rustls/src/msgs/handshake.rs

```diff
@@ -1540,6 +1582,22 @@ impl Codec<'_> for ClientEcdhParams {
     }
 }
 
+#[derive(Debug)]
+pub(crate) struct ClientDhParams {
+    pub(crate) public: PayloadU16,
+}
+
+impl Codec<'_> for ClientDhParams {
+    fn encode(&self, bytes: &mut Vec<u8>) {
+        self.public.encode(bytes);
+    }
+
+    fn read(r: &mut Reader) -> Result<Self, InvalidMessage> {
+        let pb = PayloadU16::read(r)?;
+        Ok(Self { public: pb })
```

Comment:
Style nit: please avoid the unnecessary `pb` binding by just inlining it.

Reply:
It will be a net loss in line count (i.e., more lines)😅

Reply:
Right, but I think it will be easier to read/comprehend.

Reply:
Done!